src/ray/gcs/redis_context.h

```diff
@@ -51,7 +51,7 @@ class RedisContext {
   RedisContext()
       : context_(nullptr), async_context_(nullptr), subscribe_context_(nullptr) {}
   ~RedisContext();
-  Status Connect(const std::string &address, int port, bool sharding);
+  Status Connect(const std::string &address, int port);
```

Comment:
@pcmoritz added this `sharding` flag just for the tests, so that we could run some of the tests with a single Redis shard and not always need to start up at least 2 shards.

This shouldn't have an effect on regular execution, but we may want to add it back for the purpose of simplifying some tests.

Reply:
Done